### ISSUE TITLE
Error in Gnome 3.38 after upgrade

### ISSUE THREAD

**Issue by silviobierman** [micheleg/dash-to-dock#1320]
I couple of weeks ago I installed a version from master because D2D stopped working after my Gnome update from 3.36 to 3.37. That one worked fine.
Today I got an upgrade prompt. The upgrade gave an error and now the extension no longer works. Is there a version that I can revert to?

**Comment by ChillerDragon** [comment]
Same issue here I wonder if there is a way to debug or get logs and help fixing it.

**Comment by silviobierman** [comment]
Sure. I would be happy to test things out.

I was mainly asking to check if 3.38 users where actually meant to use 
the upgrade or if it was merely an upgrade of the previous version (the 
one that we used on 3.36). If the latter is the case it would be obvious 
that this version does not work for me (yet).

I happened to have the previous version laying around so I reinstalled 
that and am on my feet again. But like I said: I would be happy to 
"upgrade" again and test/provide info.

**Comment by PeterFajner** [comment]
Try logging out and back in, that fixed it for me.

**Comment by silviobierman** [comment]
Thanks but I tried that already. It is a persistent problem for me.

**Comment by silviobierman** [comment]
I think I found out what went wrong. Upgrading to the current version (69) from the dash-to-dock-gnome-3-38 manually installed version not only crashes D2D but also disables Gnome extensions altogether. That was probably why logging of/on did not solve the problem. After manually reinstalling the earlier version I apparently re-enabled Gnome extensions.

I did a complete removal of D2D (rm -rf), logged out and back in and reinstalled from the Gnome extension site. No problem whatsoever so I consider the issue solved.

Sorry for the noise.
